src/main/kotlin/com/team4099/robot2023/subsystems/elevator/ElevatorIOKraken.kt

```diff
@@ -0,0 +1,118 @@
+        elevatorLeaderConfiguration.CurrentLimits.SupplyTimeThreshold = ElevatorConstants.LEADER_SUPPLY_TIME_THRESHOLD.inSeconds
+        elevatorLeaderConfiguration.CurrentLimits.SupplyCurrentLimitEnable = true
+        elevatorLeaderConfiguration.CurrentLimits.StatorCurrentLimit = ElevatorConstants.LEADER_STATOR_CURRENT_LIMIT.inAmperes
+        elevatorLeaderConfiguration.CurrentLimits.StatorCurrentLimitEnable = false
+
+        elevatorFollowerConfiguration.CurrentLimits.SupplyCurrentLimit = ElevatorConstants.FOLLOWER_SUPPLY_CURRENT_LIMIT.inAmperes
+        elevatorFollowerConfiguration.CurrentLimits.SupplyCurrentThreshold = ElevatorConstants.FOLLOWER_THRESHOLD_CURRENT_LIMIT.inAmperes
+        elevatorFollowerConfiguration.CurrentLimits.SupplyTimeThreshold = ElevatorConstants.FOLLOWER_SUPPLY_TIME_THRESHOLD.inSeconds
+        elevatorFollowerConfiguration.CurrentLimits.SupplyCurrentLimitEnable = true
+        elevatorFollowerConfiguration.CurrentLimits.StatorCurrentLimit = ElevatorConstants.FOLLOWER_STATOR_CURRENT_LIMIT.inAmperes
+        elevatorFollowerConfiguration.CurrentLimits.StatorCurrentLimitEnable = false
+
+        elevatorLeaderConfiguration.MotorOutput.NeutralMode = NeutralModeValue.Brake
+        elevatorFollowerConfiguration.MotorOutput.NeutralMode = NeutralModeValue.Brake
+        elevatorLeaderKraken.configurator.apply(elevatorLeaderConfiguration)
+        elevatorFollowerKraken.configurator.apply(elevatorFollowerConfiguration)
+        elevatorLeaderKraken.inverted = true
+        elevatorFollowerKraken.inverted = false
+        MotorChecker.add("Elevator", "Extension", MotorCollection(mutableListOf(
+                Falcon500(elevatorLeaderKraken, "Leader Extension Motor"), (Falcon500(elevatorFollowerKraken, "Follower Extension Motor"))), ElevatorConstants.LEADER_STATOR_CURRENT_LIMIT, 30.celsius, ElevatorConstants.LEADER_STATOR_CURRENT_LIMIT-30.amps, 110.celsius))
+        elevatorLeaderStatorCurrentSignal = elevatorLeaderKraken.statorCurrent
+        elevatorLeaderSupplyCurrentSignal = elevatorLeaderKraken.supplyCurrent
+        elevatorLeadertempSignal = elevatorLeaderKraken.deviceTemp
+        elevatorLeaderDutyCycle = elevatorLeaderKraken.dutyCycle
+        elevatorFollowerStatorCurrentSignal = elevatorFollowerKraken.statorCurrent
+        elevatorFollowerSupplyCurrentSignal = elevatorFollowerKraken.supplyCurrent
+        elevatorFollowertempSignal = elevatorFollowerKraken.deviceTemp
+        elevatorFollowerDutyCycle = elevatorFollowerKraken.dutyCycle
+    }
+
+    override fun configPID(kP: ProportionalGain<Meter, Volt>, kI: IntegralGain<Meter, Volt>, kD: DerivativeGain<Meter, Volt>) {
+        val pidController = Slot0Configs()
+        pidController.kP = leaderSensor.proportionalPositionGainToRawUnits(kP)
+        pidController.kI = leaderSensor.integralPositionGainToRawUnits(kI)
+        pidController.kD = leaderSensor.derivativePositionGainToRawUnits(kD)
```

Comment:
you need to actually apply the configuration as well. Looks like this

`m_talonFX.getConfigurator().apply(slot0Configs)`